src/sentry/tasks/post_process.py

```diff
@@ -113,7 +113,7 @@ def handle_owner_assignment(project, group, event):
 
 
 @instrumented_task(name="sentry.tasks.post_process.post_process_group")
-def post_process_group(event, is_new, is_regression, is_sample, is_new_group_environment, **kwargs):
+def post_process_group(event, is_new, is_regression, is_new_group_environment, **kwargs):
```

Comment:
I pulled out the post process changes into https://github.com/getsentry/sentry/pull/14758 like you suggested.

We want 2 to happen - worker goes first. Since this only gets called with keyword arguments, the tasks should succeed.

Reply:
"We want 2 to happen". yes but is it the way it happens in prod today ?